### DISCUSSION TITLE
Slow performance when using row actions

### DISCUSSION THREAD

**Discussion by chrisbekas** [icflorescu/mantine-datatable#550] [General]
When having row actions with ActionIcons in a column, I find the table slows down significantly when re-rendering (e.g. searching). I've tested with a table of only 200 rows and it's quite a big difference with and without the row cell actions. 

Has anyone else seen this behaviour?

**Comment by icflorescu** [comment]
Have you considered using pagination?

**Comment by chrisbekas** [comment]
@icflorescu yes have considered pagination but not a huge fan... Something like a 'load more records' would be preferable. Thanks for all your great work!
